<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.942591, y2:0.977, stop:0.164773 rgba(200, 40, 144, 255), stop:0.988636 rgba(255, 255, 255, 255))}

</string>
   </property>
   <widget class="QLabel" name="number_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>300</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 36pt &quot;Poor Richard&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Vehicle Number</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdsuper">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(3, 3, 3);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_amount">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);

font: 16pt &quot;Poor Richard&quot;;
</string>
    </property>
    <property name="text">
     <string>Enter The Ammount</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="next_Button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>440</y>
      <width>200</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: 19pt &quot;Poor Richard&quot;;
color: rgb(255, 255, 255);
background-color: rgb(163, 0, 81);</string>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QLabel" name="cust_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 36pt &quot;MS Shell Dlg 2&quot;;
font: 20pt &quot;Poor Richard&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Customer Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>781</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 36pt &quot;MS Shell Dlg 2&quot;;
font: 16pt &quot;Poor Richard&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);

font: 16pt &quot;Poor Richard&quot;;
</string>
    </property>
    <property name="text">
     <string>Weekly Allowance </string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>260</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);

font: 36pt &quot;Poor Richard&quot;;
</string>
    </property>
    <property name="text">
     <string>L</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>360</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);

font: 36pt &quot;Poor Richard&quot;;
</string>
    </property>
    <property name="text">
     <string>L</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="new_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>510</y>
      <width>200</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: 19pt &quot;Poor Richard&quot;;
color: rgb(255, 255, 255);
background-color: rgb(163, 0, 81);</string>
    </property>
    <property name="text">
     <string>Next Customer</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
